lenovo/thinkpad/general.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, ... }:
+
+with lib;
+
+{
+  hardware.trackpoint = mkDefault {
+    enable = true;
+    emulateWheel = true;
+  };
+
+  hardware.enableAllFirmware = true;
```

Comment:
Not sure if this should be a default. At least overridable (`mkDefault`). The modules should probably add hardware options more selectively.

Reply:
At the *very* least, `enableRedistributableFirmware` should be used to avoid unfree package by default.

Reply:
ah good to know. I've seen `enableAllFirmware` in other modules and it works on my ThinkPad, so I kept it for now :-)

Reply:
ok, I switched my machine to redistributable firmware only and it seems to work as well :-)